delivery/components/TelemetryEntry.yaml

```diff
@@ -10,9 +10,16 @@ TelemetryEntry:
       type: integer
       format: int64
       description: Timestamp of the entry, in milliseconds elapsed since UNIX epoch.
+    mode:
+      $ref: "./DecisioningMethod.yaml#/DecisioningMethod"
```

Comment:
I just noticed a subtlety in decisioningMode as opposed to decisioningMethod:  
From the wiki:  `The execution mode server-side, on-device or on-device-cached. On-device-cached means that the artifact is cached and we don't go over the wire.` 
So I guess we need to make a new enum with those options, inside DecisioningMode.yaml

Reply:
good catch, might have to make sure I haven't messed that up anywhere else